src/modules/launcher/Plugins/Microsoft.Plugin.Indexer/Main.cs

```diff
@@ -85,15 +84,7 @@ public List<Result> Query(Query query, bool isFullQuery)
                                 IcoPath = WarningIconPath,
                                 Action = e =>
                                 {
-                                    try
-                                    {
-                                        Process.Start(GetWindowsSearchSettingsProcessInfo());
-                                    }
-                                    catch (Exception ex)
-                                    {
-                                        Log.Exception($"Unable to launch Windows Search Settings: {ex.Message}", ex, GetType());
-                                    }
-
+                                    Helper.OpenInShell("ms-settings:cortana-windowssearch");
```

Comment:
In some cases the returned value of `Helper.OpenInShell` is not checked and no error is shown.
I haven't check all the places where this is done, consider verifying why is not done and in case add a comment motivating why the error is not shown to the user.

Reply:
Some plugins are checking the result and display a MessageBox.
Should we always display it in case of error? We can consider move the MessageBox inside `Helper.OpenInShell`.